src/braket/analog_hamiltonian_simulator/rydberg/validators/hamiltonian.py

```diff
@@ -26,20 +26,18 @@ def max_one_driving_field(cls, values):
         return values
 
     @root_validator(pre=True, skip_on_failure=True)
-    def max_one_shifting_field(cls, values):
-        shifting_fields = (
-            values["shiftingFields"]
-            if "shiftingFields" in values.keys()
-            else values["localDetuning"]
+    def max_one_local_detuning(cls, values):
+        local_detunings = (
+            values["localDetuning"] if "localDetuning" in values.keys() else values["localDetuning"]
```

Comment:
Wouldn't this always be `values["localDetuning"]`?

Reply:
Fixed